Makefile

```
@@ -46,6 +56,25 @@ $(VERSION).tar.$(SRC_EXT):
 $(subst rpm,%,$(RPMS)): $(SPEC) $(SOURCES)
 	rpmbuild -bb $(COMMON_RPM_ARGS) $(RPM_BUILD_OPTIONS) $(SPEC)
 
+$(subst deb,%,$(DEBS)): $(SPEC) $(SOURCES)
+	# Unpack tarball
+	export TAR_OPTIONS="--owner=0 --group=0 --numeric-owner"
+	mkdir -p ${DEB_BUILD}
+	tar -C $(DEB_BUILD) --strip-components=1 --atime-preserve \
+	  -xpf _topdir/SOURCES/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+	# patch tarball
+	for f in _topdir/SOURCES/*.patch; do \
```

You could just use `$(PATCHES)` here and probably should.  We can make a point that `PATCHES` MUST list them in order to be applied since they could be order dependent.  Whereas a glob of a dir might not get them in the correct order.
```suggestion
	for f in $(PATCHES); do \
```